### ISSUE TITLE
Include Artboard Background

### ISSUE THREAD

**Issue by skipjack** [zeroheight/zeroheight-sketch-plugin#5]
> Not sure if this would require plugin work or work to the core service (which I guess not open source). Anyway didn't know where else to post this request, please advise if there's a better location.

Just started using ZeroHeight, and it's worked pretty well so far! One thing I noticed is that the backgrounds set on symbol artboards...

![image](https://user-images.githubusercontent.com/8988697/29827289-8ceebf84-8ca7-11e7-86de-aa59140643bf.png)

... aren't taken into account in any of the symbol screenshots on the site:

![image](https://user-images.githubusercontent.com/8988697/29827276-853b28c2-8ca7-11e7-87cc-8338cdf16c38.png)

Is this intentional, or something you'd consider adding?

**Comment by skipjack** [comment]
I think taking this into account would make the "Light/Dark" option obsolete and give users finer grained control over the background color.

**Comment by robintindale** [comment]
yep good suggestion - I think it used to take the background in older versions of Sketch, but something must have changed :/
